XcodeServerSDK/XcodeServer.swift

```diff
@@ -106,7 +106,7 @@ public extension XcodeServer {
     }
     
     //API functionality
-    private func sendRequestWithMethod(method: HTTP.Method, endpoint: XcodeServerEndPoints.Endpoint, params: [String: String]?, query: [String: String]?, body: NSDictionary?, completion: HTTP.Completion) {
+    internal func sendRequestWithMethod(method: HTTP.Method, endpoint: XcodeServerEndPoints.Endpoint, params: [String: String]?, query: [String: String]?, body: NSDictionary?, completion: HTTP.Completion) {
```

Comment:
I would, but Xcode was yelling at me 😓 I will try again later with this as it bothers me too... 😉